packages/vue/src/components/organisms/SfAccordion/SfAccordion.html

```diff
@@ -1,3 +1,4 @@
-<div>
+<div class="sf-accordion">
+  <!--@slot Default slot for multiple SfAccordionItem elements-->
   <slot></slot>
```

Comment:
I think you should have a default markup and not only an empty slot.
My suggestion is receive a array of objects as prop, maybe named `items`, item object may have `label` and `id` properties, then you can render the list of accordion items by default, and emit an event on click passing the id as param.
Wdyt?

Reply:
`items` may also have another `items` properties (`items.items`) optionally, then you can handle multi-level

Reply:
Hi @leomp12 I agree with you regarding the first comment, I have implemented it as you described. But I think the idea from the second comment might be a little bit too much for this component. User can always use slots for that imho and implementing that functionality would be an overkill.